docs/source/notebooks_and_ipython/kedro_and_notebooks.md

```diff
@@ -92,6 +92,18 @@ When you run the cell:
  'params:example_learning_rate'
 ]
 ```
+
+#### Search datasets with regex
+If you do not remember the exact name of a dataset, you can provide a regular expression to search datasets.
+```ipython
+catalog.list("example*")
+```
+
+When you run the cell:
+
+```ipython
+['example_iris_data']
```

Comment:
Shouldn't it output these names as well `params:example_test_data_ratio`, `params:example_num_train_iter`, `params:example_learning_rate`?

<img width="1052" alt="Screenshot 2024-06-05 at 11 23 32" src="https://github.com/kedro-org/kedro/assets/114401125/14a40045-7b2e-472a-9c42-617b0b45df35">

Reply:
thanks! I should really run it in a project but it's not possible to run `pandas-iris` anymore. I just go on and update the docs with spaceflights instead.